react-native-webview-bridge.podspec

```
@@ -0,0 +1,21 @@
+equire 'json'
```

Ah, rewrote this as a Github Suggestion for easier application!
```suggestion
require 'json'
```